crates/hyperswitch_connectors/src/connectors/elavon.rs

```diff
@@ -191,7 +169,10 @@ impl ConnectorIntegration<Authorize, PaymentsAuthorizeData, PaymentsResponseData
 
         let connector_router_data = elavon::ElavonRouterData::from((amount, req));
         let connector_req = elavon::ElavonPaymentsRequest::try_from(&connector_router_data)?;
-        Ok(RequestContent::Json(Box::new(connector_req)))
+        router_env::logger::info!(raw_connector_request=?connector_req);
```

Comment:
Iti's because we are converting the `raw_connector_request` to XML using the `struct_to_xml` function, In which we are converting the complete request content into the Secret<>,therefore, we won’t be able to see the request that we sent to the connector. Hence, we are logging it before calling struct_to_xml."
```

pub fn struct_to_xml<T: Serialize>(
    item: &T,
) -> Result<HashMap<String, Secret<String, WithoutType>>, errors::ConnectorError> {
    let xml_content = quick_xml::se::to_string_with_root("txn", &item).map_err(|e| {
        router_env::logger::error!("Error serializing Struct: {:?}", e);
        errors::ConnectorError::ResponseDeserializationFailed
    })?;

    let mut result = HashMap::new();
    result.insert(
        "xmldata".to_string(),
        Secret::<_, WithoutType>::new(xml_content),
    );
    Ok(result)
}
```